Interop/Runtime/src/main/kotlin/kotlinx/cinterop/Utils.kt

```diff
@@ -390,11 +390,28 @@ private class CString(val bytes: ByteArray): CValues<ByteVar>() {
     }
 }
 
+private object EmptyCString: CValues<ByteVar>() {
+    override val size get() = 1
+    override val align get() = 1
+
+    private val placement =
+            interpretCPointer<ByteVar>(nativeMemUtils.alloc(1, 1).rawPtr)!!.also {
+                it[0] = 0.toByte()
+            }
+
+    override fun getPointer(scope: AutofreeScope): CPointer<ByteVar> {
+        return placement
+    }
+    override fun place(placement: CPointer<ByteVar>): CPointer<ByteVar> {
+        return placement
```

Comment:
Seems wrong. This method should copy the contents to the memory pointed by its parameter, and then return the parameter.